drivers/hmis/spec/requests/hmis/lookup_client_spec.rb

```diff
@@ -175,6 +176,55 @@
     expect(result.dig('data', 'client', 'image')).not_to be_nil
   end
 
+  context 'with a client who is chronically homeless per HUD definition' do
+    let!(:warehouse_ds) { create :destination_data_source }
+    let!(:client) { create :grda_warehouse_hud_client, data_source_id: warehouse_ds.id }
+    let!(:project) do
```

Comment:
I don't think this is needed, the enrollment will create its own project if you don't pass one. Or you could use p1 which already exists.